CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Starting with version `6.6.0` of the `sentry`, [Sentry's version >= v21.9.0](htt
 
 * Fix: Profiling rate decreased from 300hz to 100hz; fixed profiling traces folder creation on manual sdk init (#1997)
 * Fix: Allow disabling sending of client reports via Android Manifest and external options (#2007)
+* Ref: Upgrade Spring Boot dependency to 2.5.13 (#2011)
```

Comment:
@maciejwalkowiak or @adinauer please also add it to https://docs.sentry.io/platforms/java/migration/

Reply:
It does not require anything from users migrating to Sentry SDK 6.0